frontend/src/components/articles/articleListing.tsx

```diff
@@ -74,25 +74,52 @@ const ArticleListing: FunctionalComponent<Props> = ({ section }) => {
     let singleArticleLink = `/section/${formatUrl(article.tag)}/${article.tag_meta ? formatUrl(article.tag_meta) + '/' : ''}${formatUrl(article.title)}/${article.id}`
     return (
       <div class={style.mobileHomeArticleContainer}>
-        <div class={style.mobileTagContainer}>
-          <p class={style.mobileTag}>{article.tag}</p>
-          <div class={style.share}>
+
+        <div class={style.content}>
+          <div class={style.mobileTagContainer}>
+            <h1 class={style.mobileTag}>{article.tag}</h1>
           </div>
-        </div>
 
-        <div >
-          <Link href={singleArticleLink}>
-            <img class={style.mainImage} src={article.img_src} />
+          <Link
+            href={`/section/${article.tag
```

Comment:
For readability purposes, can you replace this whole href string with {singleArticleLink}. I actually had done that already but looks like it got deleted with the merge. 

I'm checking for other possible deletions. Let me know if it looks different when you run it locally after you had resolved the merge conflict.